pt/utils/utils.py

```diff
@@ -0,0 +1,27 @@
+import json
+import iota
+
+# from config import API_URI
+
+API_URI = 'http://node10.puyuma.org:14265'
+
```

Comment:
You should put `API_URI`  in config file.

Reply:
Fix in 530a660.